README.md

```diff
@@ -3298,6 +3549,180 @@ Touch Sensor changed event. The robot sends a Touch Sensor Event whenever one or
 
 Note that the Create 3 robot maps button 1 (·) to `FL` and button 2 (··) to `FR`.
 
+### Device 19 - Docking Sensors
```

Comment:
@shamlian FYI, this PR also includes the Docking Sensors from #18 (probably unintentional?)

Reply:
I don't know what you're talking about 😇